docs/advisors.md

```diff
@@ -18,10 +18,15 @@ As soon as you [connect your PMM instance to Percona Platform](../docs/connect-p
 
 If you are a Percona customer with a Percona Customer Portal account, you get additional access to Standard/Premium checks, which offer even more advanced database health information.
 
-Depending on the entitlements available for your Percona Account, the set of Advisor checks that PMM can download from Percona Platform differ in terms of complexity and functionality. For information about which Advisor checks are included in our subscriptions, see the [Percona Platform Subscription plans](https://www.percona.com/software/percona-platform/subscription).
+Depending on the entitlements available for your Percona Account, the set of Advisor checks that PMM can download from Percona Platform differ in terms of complexity and functionality. 
 
 ## List of Advisors
-Check out the complete list of checks and their availability for **Basic**, **Standard**, and **Premium** plans in the tables below:
+
+The **Advisors** page in Percona Platform lists the database checks that you can run from PMM on your connected instance.  
```

Comment:
I would suggest removing 'on your connected instance' because we also show those basic checks available if the instance is not connected.
We can also add that if the Percona Platform account is not fully set up (user hasn't any organization and PMM is not connected) we suggest they perform these steps in order to receive a bigger amount of checks that PMM can download from Percona Platform.

Reply:
removed " on your connected instance". 
The second bit is mentioned in the **How to get more Advisors** section.